ProjectVoyage/View Controllers/Chat list/ChatCreateViewController.m

```diff
@@ -0,0 +1,94 @@
+//
+//  ChatCreateViewController.m
+//  ProjectVoyage
+//
+//  Created by Gui David on 7/19/22.
+//
+
+#import "ChatCreateViewController.h"
+#import "Chat.h"
+#import "Util.h"
+
+@interface ChatCreateViewController ()
+@property (nonatomic, strong) IBOutlet UITextField *recipientNameTextField;
+@property (nonatomic, strong) IBOutlet UITextField *chatDescriptionTextField;
+@property (nonatomic, strong) IBOutlet UILabel *dateLabel;
+@property (nonatomic, strong) IBOutlet UIImageView *recipientImage;
+
+@end
+
+@implementation ChatCreateViewController 
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [Util roundImage:_recipientImage];
+}
+
+- (IBAction)didPressCreate:(id)sender {
+    // GD view controller not being dismissed
+    [self dismissViewControllerAnimated:TRUE completion:^{
+        [self postChat];
```

Comment:
TRUE -> YES

I don't think this works. Once this view controller is dismissed, you aren't guaranteed to have _recipientNameTextField or _chatDescriptionTextField (any of the ivars) available to you.